src/main/java/convobot/utils/Storage.java

```diff
@@ -0,0 +1,70 @@
+package utils;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+
+import tasks.Task;
+
+/**
+ * The {@code Storage} class provides methods for reading and writing task data to a file.
+ * It interacts with the file system to perform read and write operations on the task data file.
+ */
+public class Storage {
+
+    private final String filePath;
```

Comment:
Could this be named in uppercase as a constant? Like FILE_PATH?

Reply:
Accepted.